### PR TITLE
Added problem 61 in project euler

### PR DESCRIPTION
**Pull request by Utshaan**
Added the implementation to problem 60 in project_euler/.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: project_euler/problem_060/sol1.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Solution:
+    Simply check all prime numbers till the limit.
+
+"""
+
+from math import floor, sqrt
+
+
+def primes_till(x: int) -> set[int]:
```
[the remark this answers is not in the document]
Comment (Utshaan) [reply project_euler/problem_060/sol1.py]: Changed it to `number`

File: project_euler/problem_060/sol1.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    if x >= 2:
+        for i in range(2, floor(sqrt(x) + 1)):
+            if x % i == 0:
+                return False
+        return True
+    else:
+        return False
+
+
+def prime_pair_test(a: int, b: int) -> bool:
```
[the remark this answers is not in the document]
Comment (Utshaan) [reply project_euler/problem_060/sol1.py]: changed them to `number1` and `number2`.

**Review by dhruvmanila — COMMENTED** [review]
The solution takes 32 seconds! Maybe we could profile the code and optimize it? https://github.com/TheAlgorithms/Python/actions/runs/3253183598/jobs/5340202639#step:5:21

**Comment by Utshaan** [comment]
@dhruvmanila I don't think I can figure out how to profile it furthur. I have added the solution to problem 61 though, how can I remove problem 60 from this pr?

**Pull request closed without merging**